app/views/schools/cohort_setup/expect_any_ects.html.erb

```diff
@@ -1,4 +1,4 @@
-<% title = "Does your school expect any new ECTs in the new academic year?" %>
+<% title = "Will any ECTs start their induction at your school in the 2023 to 2024 academic year?" %>
```

Comment:
We should probably add some code here to show the right academic year - otherwise we’ll forget to change it next year... 😄

@tonyheadford would you mind helping us with this?

Reply:
Ah, I think I figured this out in e63c037399f3f565f5ddfe76349e323dc55b0be7